tests/phpunit/unit-tests/Helper/Fields/Test_Field_Repeater.php

```diff
@@ -0,0 +1,48 @@
+<?php
+declare( strict_types=1 );
+
+namespace GFPDF\Helper\Fields;
+
+use WP_UnitTestCase;
+use GFAPI;
+use GPDFAPI;
+
+/**
+ * @package     Gravity PDF
+ * @copyright   Copyright (c) 2022, Blue Liquid Designs
+ * @license     http://opensource.org/licenses/gpl-2.0.php GNU Public License
+ */
+
+/**
+ * Class Test_Field_Repeater
+ *
+ *
+ * @group   helper
+ * @group   fields
+ */
+class Test_Field_Repeater extends WP_UnitTestCase {
+
+	/**
+	 * Test the Gravity PDF Repeater field add_entry_meta method
+	 */
+	public function test_repeater_add_entry_meta_method() {
+		$form  = json_decode( trim( file_get_contents( dirname( __FILE__ ) . '/../../json/repeater-consent-form.json' ) ), true );
+		$entry = json_decode( trim( file_get_contents( dirname( __FILE__ ) . '/../../json/repeater-consent-entry.json' ) ), true );
+
+		$form_id          = GFAPI::add_form( $form );
+		$entry['form_id'] = $form_id;
+		$entry_id         = GFAPI::add_entry( $entry );
+		$form_data        = GPDFAPI::get_form_data( $entry_id );
+
+		$repeater = new Field_Repeater( new \GF_Field_Repeater( $form["fields"][1] ), GFAPI::get_entry( $entry_id ), GPDFAPI::get_form_class(), GPDFAPI::get_misc_class() );
+		$items    = $repeater->add_entry_meta( $form_data['repeater'][999][0][15] );
```

Comment:
This is a bad unit test. For example, you don't know what the structure of `$form_data['repeater'][999][0][15]` is and you cannot verify the `add_entry_meta()` does what it is meant to do. Further, you aren't testing the value of the array keys to ensure the array replace goes as expected. 

The concern is the `array_replace` overrides the wrong array keys and returns the wrong data, and this is where well-written tests verify the functionality.

Along with passing in different data to `add_entry_meta`, different `$entry` objects should be tested as well.